examples/external-oauth/jupyterhub_config.py

```diff
@@ -1,5 +1,16 @@
 import os
 
+# Allow anyone to authenticate
```

Comment:
In some other examples that have this, I've [put it at the bottom](https://github.com/jupyterhub/jupyterhub/blob/f4163069139251c2be2d7752607518e97bd31079/examples/server-api/jupyterhub_config.py#L54-L57) to separate it as clearly the "this isn't relevant to the topic of the example, but is needed to make the example runnable". I think that makes it easier to "stop reading here..."

Reply:
@minrk I've moved it to the bottom!